ReSwift/CoreTypes/Store.swift

```diff
@@ -29,7 +33,7 @@ public class Store<State: StateType>: StoreType {
 
     private var reducer: AnyReducer
 
-    var subscribers: [AnyStoreSubscriber] = []
+    var subscribers: [(subscriber: AnyStoreSubscriber, selector:(State -> Any)?)] = []
```

Comment:
Colon Violation: Colons should be next to the identifier when specifying a type. (colon)